test-mktemp

```diff
@@ -53,7 +53,7 @@ MKTEMP='mktemp -p /tmp/ $1'
 testfun temp.XXXXXX
 
 # FreeBSD way
-MKTEMP="mktemp -t /tmp"
+MKTEMP="mktemp -t $1"
```

Comment:
Here `$1` would be resolved immediately (as an empty string). Did you intend this?